tests/data/fixtures/db.yaml

```diff
@@ -1,3 +1,7 @@
+- model: sites.site
```

Comment:
let's make this clean and only commit the changes that are being made by adding this app